filebeat/channel/outlet.go

```diff
@@ -42,7 +43,7 @@ func (o *Outlet) SetSignal(signal <-chan struct{}) {
 	o.signal = signal
 }
 
-func (o *Outlet) OnEvent(event *input.Event) bool {
+func (o *Outlet) OnEvent(event *input.Data) bool {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Outlet.OnEvent should have comment or be unexported